integration-libs/digital-payments/src/utils/dp-constants.ts

```diff
@@ -0,0 +1,3 @@
+export const CURRENT_USER = 'current';
```

Comment:
You can find these variables in @spartacus/core under `OCC_CART_ID_CURRENT` and `OCC_USER_ID_CURRENT`.